pkg/config/templates/rancherd-10-harvester.yaml

```diff
@@ -20,6 +20,45 @@ bootstrapResources:
       release: rancher-monitoring
     name: rancher-monitoring-operator
     namespace: cattle-monitoring-system
+- apiVersion: helm.cattle.io/v1
+  kind: HelmChartConfig
+  metadata:
+    name: rke2-ingress-nginx
+    namespace: kube-system
+  spec:
+    valuesContent: |-
+      controller:
+        admissionWebhooks:
+          port: 8444
+        publishService:
+          pathOverride: kube-system/ingress-expose
+        extraArgs:
+          default-ssl-certificate: cattle-system/tls-rancher-internal
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    name: tls-ingress
+    namespace: cattle-system
+  type: Opaque
+  data:
+    tls.crt: Cg==
+    tls.key: Cg==
```

Comment:
Q: is pre-render the empty `tls-ingress` secret required?

Reply:
We can either create it here or create it on the Harvester server side.

Reply:
both works for me, we can leave as what it is, thanks.